aspnetcore/tutorials/publish-to-azure-webapp-using-vs.md

```diff
@@ -14,187 +14,170 @@ uid: tutorials/publish-to-azure-webapp-using-vs
 
-![Internal Server Error: A database operation failed while processing the request. SQL exception: Cannot open the database. Applying existing migrations for Application DB context may resolve this issue.](publish-to-azure-webapp-using-vs/_static/mig.png)
+* Click the **About** and **Contact** links to verify the web application works.
 
-You can fix the problem in two different ways:
+![Web application open in Microsoft Edge on localhost](publish-to-azure-webapp-using-vs/_static/show.png)
 
-* Tap **Apply Migrations** and, once the page updates, refresh the page; or
+* Select **Register** and register a new user. You can use a fictitious email address. When you submit, the page displays the following error:
 
-* Run the following from a command prompt in the project's directory:
+    *"Internal Server Error: A database operation failed while processing the request. SQL exception: Cannot open the database. Applying existing migrations for Application DB context may resolve this issue."*
 
-  <!-- literal_block {"ids": [], "xml:space": "preserve"} -->
+* Select **Apply Migrations** and, once the page updates, refresh the page.
 
-  ```
-  dotnet ef database update
-     ```
+![Internal Server Error: A database operation failed while processing the request. SQL exception: Cannot open the database. Applying existing migrations for Application DB context may resolve this issue.](publish-to-azure-webapp-using-vs/_static/mig.png)
 
-The app displays the email used to register the new user and a **Log off** link.
+The app displays the email used to register the new user and a **Log out** link.
 
-![Web application open in Microsoft Edge. The Register link is replaced by the text Hello abc@example.com!](publish-to-azure-webapp-using-vs/_static/hello.png)
+![Web application open in Microsoft Edge. The Register link is replaced by the text Hello email@domain.com!](publish-to-azure-webapp-using-vs/_static/hello.png)
 
 ## Deploy the app to Azure
 
-Confirm that the published app for deployment isn't running. Files in the *publish* folder are locked when the app is running. Deployment can't occur because locked files can't be copied.
+Close the web page, return to Visual Studio, and select **Stop Debugging** from the **Debug** menu.
 
 Right-click on the project in Solution Explorer and select **Publish...**.
 
 ![Contextual menu open with Publish link highlighted](publish-to-azure-webapp-using-vs/_static/pub.png)
 
-In the **Publish** dialog, tap **Microsoft Azure App Service**.
+In the **Publish** dialog, select **Microsoft Azure App Service** and click **Publish**.
 
 ![Publish dialog](publish-to-azure-webapp-using-vs/_static/maas1.png)
 
-Tap **New...** to create a new resource group. Creating a new resource group will make it easier to delete all the Azure resources you create in this tutorial.
+* Name the app a unique name. 
 
-![App Service dialog](publish-to-azure-webapp-using-vs/_static/newrg1.png)
+* Select an MSDN subscription.
```

Comment:
Select an ~~MSDN~~ Azure subscription.

Reply:
Perhaps change it simply to "Subscription" since that is what the dialog says? My VS UI has it labelled that way but specifically lists MSDN subscriptions and not the Azure ones.